prow/getting_started_deploy.md

```diff
@@ -261,6 +261,25 @@ label. When you make a change to the plugin config and push it with `make
 update-plugins`, you do not need to redeploy any of your cluster components.
 They will pick up the change within a few minutes.
 
+### Set namespaces for prowjobs and test pods
+
+Add the following to `config.yaml`:
+
+```yaml
+prowjob_namespace: default
+pod_namespace: test-pods
```

Comment:
we might want to put these to `starter.yaml` too, or it may cause the initial `echo-test` jobs not fully working